common/config/loader.go

```diff
@@ -176,6 +380,15 @@ func file(name string, suffix string) string {
 	return name + "." + suffix
 }
 
-func path(dir string, file string) string {
-	return dir + "/" + file
+func getEnvMap() map[string]string {
```

Comment:
nit: don't use `get` for getters in Go.

```suggestion
func loadEnvMap() map[string]string {
```